dom/lists/DOMTokenList-supports.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>DOMTokenList.supports</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+"use strict";
+
+var tokens = {
+  // https://html.spec.whatwg.org/multipage/interaction.html#the-dropzone-attribute:concept-supported-tokens
+  el_dropzone: [
+    "copy", "move", "link", "string:test1", "string:test2", "file:test1", "file:test2"
+  ],
+  // https://html.spec.whatwg.org/multipage/embedded-content.html#the-iframe-element:concept-supported-tokens
+  iframe_sandbox: [
+    "allow-forms", "allow-modals", "allow-pointer-lock", "allow-popups",
+    "allow-popups-to-escape-sandbox", "allow-same-origin", "allow-scripts", "allow-top-navigation"
+  ],
+  // https://html.spec.whatwg.org/multipage/semantics.html#links-created-by-a-and-area-elements:concept-supported-tokens
+  a_area_rel: [
+    "noreferrer", "noopener", "prefetch"
+  ],
+  // https://html.spec.whatwg.org/multipage/semantics.html#the-link-element:concept-supported-tokens
+  link_rel: [
+    "alternate", "icon", "pingback", "prefetch", "stylesheet", "next"
```

Comment:
I suppose it would be good to clarify this in the spec. Also the bit about `next` being a synonym for `prefetch`.

http://software.hixie.ch/utilities/js/live-dom-viewer/saved/4152
In Chromium I get
```
log: alternate
log: icon
log: next
log: prefetch
log: stylesheet
log: preload
log: preconnect
log: dns-prefetch
```

The use case for this is to test support for things you would want to polyfill when there isn't native support, which hasn't been interesting until the preload/prefetch etc keywords. Testing things like `icon` is probably not so interesting. Maybe the spec should say for each keyword if it is a candidate for returning `true` in `supports()`? cc @yoavweiss